src/enums/theme.ts

```diff
@@ -0,0 +1,5 @@
+export enum CustomConsentManagerStyle {
+  P = 'p',
```

Comment:
What is this p?

Reply:
paragraph `<p`

Reply:
Oh! Maybe could add a comment in the code?